code/modules/mob/living/living_verbs.dm

```diff
@@ -187,9 +183,7 @@
 				for(var/mob/O in viewers(CM))
 					O.show_message(text("<span class='danger'>[] is trying to break [HC]!</span>", CM), 1)
 				spawn(0)
-					if(do_after(CM, 50, FALSE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
same as above.

Reply:
You shouldn't change functionality here